Mathlib/LinearAlgebra/FreeModule/Rank.lean

```diff
@@ -123,3 +137,109 @@ theorem rank_tensorProduct' (N : Type v) [AddCommGroup N] [Module R N] [Module.F
 #align rank_tensor_product' rank_tensorProduct'
 
 end CommRing
+
+section DivisionRing
+
+variable (K : Type u) [DivisionRing K]
+
+/-- Key lemma towards the Erdős-Kaplansky theorem from https://mathoverflow.net/a/168624 -/
+theorem rank_pi_nat : max ℵ₀ #K ≤ Module.rank K (ℕ → K) := by
```

Comment:
since it only proves one inequality, shouldn't it be called le_rank_pi_nat or something like that?

Reply:
I think so. Let me change the name to `max_aleph0_mk_le_rank_function_nat` (I realized the non-dependent function type is called `function` in mathlib more often than `pi`).